library/ctr_drbg.c

```diff
@@ -386,6 +336,52 @@ int mbedtls_ctr_drbg_reseed( mbedtls_ctr_drbg_context *ctx,
     return( ret );
 }
 
+int mbedtls_ctr_drbg_seed( mbedtls_ctr_drbg_context *ctx,
```

Comment:
This function lacks the needed documentation now.

Reply:
What documentation? If you mean the comments relating the implementation to the NIST standard that are present in the mbed-crypto and 2.16 PRs, they were added after 2.7 and never backported, and backporting them would be out of scope here.